autopush/web/dockerflow.py

```diff
@@ -0,0 +1,42 @@
+"""DockerFlow endpoints"""
+
+import cyclone.web
+
+from twisted.internet.threads import deferToThread
+from autopush.web.base import BaseWebHandler
+
+
+class VersionHandler(BaseWebHandler):
```

Comment:
let's have VersionHandler call self.set_header("Content-Type", "application/json")

Also let's have both new handlers override authenticate_peer_cert to do nothing (skipping auth checks) like the handlers in web.health/memusage